src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/ProjectInspector.java

```diff
@@ -276,14 +281,21 @@ public void run() {
                 }
             }
             // Add the next steps
-            cloneRepo
+           
+
+            if (!RepairnatorConfig.getInstance().isStaticAnalysis()) {
```

Comment:
avoid if/then or switch case, prefer polymorphism instead with the strategy pattern instead.